zip-0311.rst

```diff
@@ -2,7 +2,262 @@
 
   ZIP: 311
   Title: Sapling Payment Disclosure
-  Owners: Deirdre Connolly <deirdre@zfnd.org>
-  Status: Reserved
+  Owners: Jack Grigg <jack@electriccoin.co>
+  Status: Draft
   Category: Standards / RPC / Wallet
   Discussions-To: <https://github.com/zcash/zips/issues/387>
+
+
+Abstract
+========
+
+This ZIP describes a mechanism and format for disclosing information about Sapling spends
+and outputs within a transaction. In the typical case, this means enabling a sender to
+present a proof that they transferred funds to a recipient's Sapling address.
+
+
+Motivation
+==========
+
+Payment disclosure is useful in a number of situations, for example:
+
+- A sender may need to prove that their payment was sent and received by a recipient.
```

Comment:
Surely a sender can only prove that a payment is available to be received (following the protocol), rather than that it was actually received?